### PR TITLE
Omnibus: Bundle only python2 

### PR DESCRIPTION
**Pull request by albertvaka**
Added `PYTHON_RUNTIMES` to `release.json` where you can specify either `"2"`, `"3"` or both `"2,3"`. 

Nightlies from now on will only have python 2.

### CODE REVIEW

**Comment by albertvaka** [comment]
[[6.12] Remove the python 3 bits from the omnibus packages](https://trello.com/c/PxTK0CqX/128-612-remove-the-python-3-bits-from-the-omnibus-packages)

File: omnibus/config/projects/agent.rb [hunk 1]
```diff
@@ -118,16 +118,26 @@
 # Datadog agent
 dependency 'datadog-agent'
 
+python_runtimes = ENV['PYTHON_RUNTIMES'].split(',')
```
Comment (masci) [inline omnibus/config/projects/agent.rb]: let's make this a project-level variable with a default value, something like
```
python_runtimes ENV['PYTHON_RUNTIMES'].nil? ? ['2'] : ENV['PYTHON_RUNTIMES'].split(',')
```
having a default value here is more robust and makes possible bootstrapping the build by just invoking `omnibus`

File: omnibus/config/projects/agent.rb [hunk 2]
```diff
@@ -118,16 +118,26 @@
...
+
 # Additional software
 if windows?
-  dependency 'cacerts_py2'
-  dependency 'cacerts_py3'
+  dependency 'cacerts_py2' if python_runtimes.include? "2"
```
Comment (masci) [inline omnibus/config/projects/agent.rb]: if we go down the path of the project variable, this use case would be `if project.python_runtimes.include? "2"` (it applies to any software def below)

File: omnibus/config/software/datadog-agent-finalize.rb [hunk 3]
```diff
@@ -95,8 +95,12 @@
             command "find #{install_dir}/embedded '(' -name '*.pyc' -o -name '*.pyo' ')' -type f -delete -print >> #{install_dir}/embedded/.py_compiled_files.txt"
 
             # Setup pip aliases: `/opt/datadog-agent/embedded/bin/pip` will default to `pip2`
-            delete "#{install_dir}/embedded/bin/pip"
-            link "#{install_dir}/embedded/bin/pip2", "#{install_dir}/embedded/bin/pip"
+            python_runtimes = ENV['PYTHON_RUNTIMES'].split(',')
+            if python_runtimes.include? "2"
+                delete "#{install_dir}/embedded/bin/pip"
+                link "#{install_dir}/embedded/bin/pip2", "#{install_dir}/embedded/bin/pip"
+            end
```
Comment (masci) [inline omnibus/config/software/datadog-agent-finalize.rb]: we can probably add an `else` and link `pip` to `pip3` as a fall back

**Review by masci — CHANGES_REQUESTED** [review]
LGTM overall, left a couple of comments

File: omnibus/config/software/datadog-agent-finalize.rb (hunk 3 above)
Comment (albertvaka) [reply omnibus/config/software/datadog-agent-finalize.rb]: If only python3 is installed we know for sure that `pip` already points to `pip3`.

File: omnibus/config/software/datadog-a7-py2.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name "datadog-a7-py2"
+default_version "0.0.5"
```
Comment (ofek) [inline omnibus/config/software/datadog-a7-py2.rb]:
```suggestion
default_version "0.0.6"
```

File: omnibus/config/software/datadog-a7-py3.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+name "datadog-a7-py3"
+default_version "0.0.5"
```
Comment (ofek) [inline omnibus/config/software/datadog-a7-py3.rb]:
```suggestion
default_version "0.0.6"
```

**Review by ofek — CHANGES_REQUESTED** [review]

File: omnibus/config/projects/agent.rb (hunk 1 above)
Comment (albertvaka) [reply omnibus/config/projects/agent.rb]: To add project-level variables we need to change https://github.com/DataDog/omnibus-software/blob/master/lib/omnibus-software.rb

I will add the default to the current code.

**Review by masci — DISMISSED** [review]

**Review by ofek — APPROVED** [review]

**Review by masci — APPROVED** [review]

**Review by derekwbrown — APPROVED** [review]

**Pull request merged**